tests/admin_scripts/management/commands/with_suppress_command.py

```diff
@@ -0,0 +1,15 @@
+from django.core.management import BaseCommand
+
+
+class Command(BaseCommand):
+
+    help = 'Test basic commands with suppress'
+    requires_system_checks = []
+    suppressed_base_arguments = {'--verbosity', '--traceback', '--settings', '--pythonpath', '--no-color',
+                                 '--force-color', '--version'}
```

Comment:
Use hanging indentation per [Python coding style](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style):
```suggestion
    suppressed_base_arguments = {
        '--verbosity', '--traceback', '--settings', '--pythonpath',
        '--no-color', '--force-color', '--version',
    }
```